gradio/components.py

```diff
@@ -27,6 +27,7 @@
 import PIL.ImageOps
 from ffmpy import FFmpeg
 from markdown_it import MarkdownIt
+from mdit_py_plugins.dollarmath import dollarmath_plugin
```

Comment:
Do we need to add anything to `requirements.txt`?

Reply:
Never mind

Reply:
pretty sure it comes with `markdown_it` automatically